src/components/Filters/Filters.js

```diff
@@ -48,12 +48,8 @@ const Filters = ({
   selectedRecordId,
 }) => {
   const [filterPaneIsVisible, setFilterPaneIsVisible] = useState(true);
-  const [storedFilter, setStoredFilter] = useState(
-    localStorage.getItem('fincSelectFilterFilters') ? JSON.parse(localStorage.getItem('fincSelectFilterFilters')) : defaultFilter
-  );
-  const [storedSearchString, setStoredSearchString] = useState(
-    localStorage.getItem('fincSelectFilterSearchString') ? JSON.parse(localStorage.getItem('fincSelectFilterSearchString')) : defaultSearchString
-  );
+  const [storedFilter, setStoredFilter] = useState(localStorage.getItem('fincSelectFilterFilters') ? JSON.parse(localStorage.getItem('fincSelectFilterFilters')) : defaultFilter);
+  const [storedSearchString, setStoredSearchString] = useState(localStorage.getItem('fincSelectFilterSearchString') ? JSON.parse(localStorage.getItem('fincSelectFilterSearchString')) : defaultSearchString);
```

Comment:
And can we please not reformat any code that had no changes made to it?
This will help keep PRs more readable.
Thanks!